x-pack/test/api_integration/apis/ml/trained_models/get_model_stats.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common_api';
+
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  describe('GET trained_models/_stats', () => {
+    before(async () => {
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.api.createdTestTrainedModels('regression', 2);
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    it('returns trained model stats by id', async () => {
+      const { body } = await supertest
+        .get(`/api/ml/trained_models/dfa_regression_model_n_0/_stats`)
+        .auth(USER.ML_VIEWER, ml.securityCommon.getPasswordForUser(USER.ML_VIEWER))
+        .set(COMMON_REQUEST_HEADERS)
+        .expect(200);
+
+      expect(body.count).to.eql(1);
+      expect(body.trained_model_stats[0].model_id).to.eql('dfa_regression_model_n_0');
+    });
+
+    it('returns 404 if requested trained model does not exist', async () => {
+      await supertest
+        .get(`/api/ml/trained_models/not_existing_model/_stats`)
+        .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+        .set(COMMON_REQUEST_HEADERS)
+        .expect(404);
+    });
+
+    it('return an error for unauthorized user', async () => {
```

Comment:
super duper nit: should this be `returns`?

Reply:
Fixed in https://github.com/elastic/kibana/pull/104819/commits/407294afab4bd42e29d5d806f928d14b296f5490